api/v1/common.go

```diff
@@ -227,22 +233,32 @@ type DisplayTemplate interface {
 
 // +k8s:deepcopy-gen=false
 type TestFunction interface {
-	GetTestFunction() Template
+	GetTestTemplate() Template
+	GetTestThreshold() *TestThreshold
 }
 
 // +k8s:deepcopy-gen=false
 type Transformer interface {
 	GetTransformer() Template
 }
 
+type TestTemplate struct {
+	Template       `yaml:",inline" json:",inline"`
+	*TestThreshold `yaml:",inline" json:",inline"`
```

Comment:
@adityathebe Can we move this to a test.threshold field?

Reply:
and also align specify the severity:

* info = 0 
* low
* medium
* high
* critical